config/nrfconnect/chip-module/CMakeLists.txt

```diff
@@ -28,6 +28,7 @@
 if (CONFIG_CHIP)
 
 include(ExternalProject)
+include(../../zephyr/ota-image.cmake)
```

Comment:
Nit: don't know if ota-image is good name. maybe it should be more specific. matter-ota-image.cmake ?

Reply:
Given that the module is inside the Matter core directory, can it be confused with other OTA image? If so, I can rename it, sure.

Reply:
Indeed you are right. I was just looking on the path and `zephyr/ota-image.cmake` just leads to a different association in my head. I was thinking that this `ota-image.cmake` was related to `zephyr`.